test/unit/calculators/minimum_wage_calculator_test.rb

```diff
@@ -90,8 +90,8 @@ class MinimumWageCalculatorTest < ActiveSupport::TestCase
       end
 
       context "for age for living wage" do
-        should "not accept ages below 25" do
-          assert_not @calculator.valid_age_for_living_wage?(24)
+        should "not accept ages below 23" do
+          assert_not @calculator.valid_age_for_living_wage?(22)
         end
 
         should "accept ages 25 or above" do
```

Comment:
I know this test passes, but I think it should be updated to `accept ages 23 or above`